tf_encrypted/protocol/pond.py

```diff
@@ -3512,3 +3510,22 @@ def _cast_backing_public(
             y_on_1 = x_on_1.cast(backing_dtype)
 
         return PondPublicTensor(prot, y_on_0, y_on_1, x.is_scaled)
+
+
+def _cast_backing_private(prot: Pond, x: PondPrivateTensor, backing_dtype) -> PondPrivateTensor:
```

Comment:
Wonder if this should be moved to securenn if its not being used elsewhere.

Reply:
yeah I like that, not least since it's half-baked for now. will move both public and private casting

Reply:
done